jcl/src/openj9.dtfj/share/classes/com/ibm/dtfj/javacore/parser/j9/section/thread/ThreadTagParser.java

```diff
@@ -95,18 +98,33 @@ private void initThreadInfoTag() {
 		ILineRule lineRule = new ThreadInfoLineRule();
 		addTag(IThreadTypes.T_3XMTHREADINFO, lineRule);
 	}
-	
-	
+
 	/**
-	 * 
 	 *
 	 */
+	private void initJavaLThrCCL() {
+		ILineRule lineRule = new ThreadInfoLineRule() {
+			public void processLine(String source, int startingOffset) {
+				// 3XMJAVALTHRCCL            jdk/internal/loader/ClassLoaders$AppClassLoader(0x000000072254DA00)
+				consumeUntilFirstMatch(CommonPatternMatchers.whitespace);
+				addAllCharactersAsTokenAndConsumeFirstMatch(IThreadTypes.CONTEXT_CLASSLOADER_OBJECT_FULL_JAVA_NAME, CommonPatternMatchers.open_paren);
+				addPrefixedHexToken(IThreadTypes.CONTEXT_CLASSLOADER_OBJECT_ADDRESS);
+			}
+		};
+		addTag(IThreadTypes.T_3XMJAVALTHRCCL, lineRule);
+	}
+
+
+	/**
+	 * Parses
```

Comment:
I appreciate you adding to these comments, but this (and the additions below) don't help much.

Reply:
The line ending check complained about trailing spaces, so I removed them, and added a minimal comment.